go/oasis-node/cmd/common/flags/flags.go

```diff
@@ -19,10 +19,12 @@ const (
 	// CfgConsensusValidator is the flag used to opt-in to being a validator.
 	CfgConsensusValidator = "consensus.validator"
 
-	cfgVerbose = "verbose"
-	cfgForce   = "force"
-	cfgRetries = "retries"
-	cfgEntity  = "entity"
+	cfgVerbose       = "verbose"
+	cfgForce         = "force"
+	cfgRetries       = "retries"
+	cfgEntity        = "entity"
+	cfgFileDir       = "entity.file.dir"
```

Comment:
The prefix should either be `signer.` or `entity.signer.` as `entity.file.dir` sounds really confusing? Also below.